type-definitions/most.d.ts

```diff
@@ -29,7 +29,7 @@ export interface ScheduledTask {
 export interface Scheduler {
   now(): number;
   asap(task: Task): ScheduledTask;
-  delay(task: Task): ScheduledTask;
+  delay(delay: number, task: Task): ScheduledTask;
```

Comment:
LGTM.  Periodic below also needs a `period: number` param.  Could you add that one as well?